NEWS.adoc

```diff
@@ -6,6 +6,59 @@
     textures.  These are named after and in honor of Esa Repo (Espi).
   * A STARBR1 texture is now included as a counterpart to STARBR2.
```

Comment:
This HEAD section can be removed now that its contents are reflected below.

Reply:
I agree. Deleted. It seems odd that we had an `HEAD` section when there's no reason that we wouldn't release any news worthy enough to be in the file.